addons/infinite-scroll/global.css

```diff
@@ -0,0 +1,8 @@
+.djangobb tbody th {
+  border-top-style: solid;
+  border-top-width: 1px;
+}
+
+.djangobb .pagination {
+  display: none;
+}
```

Comment:
This will also hide page numbers in forum topics. It should only hide numbers in `https://scratch.mit.edu/discuss/*/` (trailing slash important)